lib/board.rb

```diff
@@ -82,6 +82,12 @@ def check_static(parsed_coordinates)
 
   def place(ship, coordinates)
     # iterate over each coordinate pair and #place_ship using @cells hash
+    #Add an all statement here to confirm that all coordiantes pass valid_coordinate
+    if valid_placement?(ship, coordinates)
```

Comment:
I like that you used our `valid_placement?` method to check whether an object could actually be placed. Didn't think of that!